opensrp-chw-core/src/main/java/org/smartregister/chw/core/custom_views/NavigationMenu.java

```diff
@@ -624,12 +647,18 @@ public void setSelectedView(String selectedView) {
 
         boolean hasSyncStatusProgressBar();
 
+        boolean hasMultipleLanguages();
+
         Intent getStockReportIntent(Activity activity);
 
         Intent getServiceReportIntent(Activity activity);
 
         String childNavigationMenuCountString();
 
         Intent getHIA2ReportActivityIntent(Activity activity);
+
+        default AlertDialog doLogOutDialog(Activity activity) {
+            return null;
```

Comment:
Why is this returning null ?

Reply:
By default it was set to null, to not show any log out dialog...but could be extended in inheriting classes for a custom pop up